vespaclient-container-plugin/src/main/java/com/yahoo/vespa/http/server/ClientFeederV3.java

```diff
@@ -184,8 +184,8 @@ private int getOverloadReturnCode(HttpRequest request) {
             try {
                 message = getNextMessage(operationId.get(), requestInputStream, settings);
             } catch (Exception e) {
-                if (log.isLoggable(LogLevel.DEBUG)) {
-                    log.log(LogLevel.DEBUG, Exceptions.toMessageString(e), e);
+                if (log.isLoggable(LogLevel.WARNING)) {
+                    log.log(LogLevel.WARNING, e.toString());
```

Comment:
Why not use `Exceptions.toMessageString()`? It will also include the message from any nested exceptions if present, but ignores stack trace.